model/browser/events.yaml

```diff
@@ -52,3 +52,92 @@ groups:
           brief: A unique ID representing this particular metric instance.
           requirement_level: required
           examples: ["v3-1677874579383-6381583661209"]
+  - id: event.enduser.user_action
+    stability: development
+    type: event
+    name: enduser.user_action
+    brief: >
+      This event describes actions performed by the user such as click, scroll, zoom, resize, etc.
+    body:
+      id: enduser.user_action
+      requirement_level: required
+      stability: development
+      type: map
+      fields:
+        - id: element
+          type: string
+          stability: development
+          brief: Target element tag name and it is obtained via `event.target.tagName`.
+          requirement_level: recommended
+          examples: ["button"]
+        - id: xpath
```

Comment:
`xpath`, `element` and `tags` feel browser specific, while the event name `enduser.user_action` makes it sound generic

Reply:
updated `enduser.user_action` to `browser.user_action`